src/views/movie_list_view.js

```diff
@@ -0,0 +1,98 @@
+import Backbone from 'backbone';
+import Movie from 'models/movie';
+import MovieView from 'views/movie_view';
+import Search from 'models/search';
+
+const MovieListView = Backbone.View.extend({
+  initialize(params){
+    this.template = params.template;
+  },
+
+  events: {
+    'click button.btn-show': 'getRentalMovies',
+    'click button.btn-search': 'getIndividualMovie',
+  },
+
+  getRentalMovies() {
+    event.preventDefault()
```

Comment:
In order to be able to call `event.preventDefault()` on this line we need to have an `event` variable. We can get one by making it a parameter for this function:
```javascript
getRentalMovies(event) {
  event.preventDefault();
  // ...
},
```